main.sh

```diff
@@ -107,40 +99,40 @@ function xml_dump {
 
 function all_sch { # arg($1:group)
     # is loaded by default so data must be loaded
-    data=($(sed 's/://g' $SCHEDULE))
```

Comment:
ए चिप्लिएछ, no i didn't , you got what i mean to say! :stuck_out_tongue_winking_eye: